script.js

```diff
@@ -0,0 +1,41 @@
+let count = 0;
+let intervalId;
+
+let startButton = document.getElementById("start");
+let pauseButton = document.getElementById("pause");
+let resumeButton = document.getElementById("resume");
+let stopButton = document.getElementById("stop");
+let countSpan = document.getElementById("count");
+
+startButton.addEventListener("click", function () {
+  intervalId = setInterval(function () {
+    count++;
+    countSpan.textContent = count;
+  }, 1000);
+  startButton.disabled = true;
+  pauseButton.disabled = false;
+});
+
+pauseButton.addEventListener("click", function () {
+  clearInterval(intervalId);
+  pauseButton.disabled = true;
+  resumeButton.disabled = false;
+});
+
+resumeButton.addEventListener("click", function () {
+  intervalId = setInterval(function () {
```

Comment:
Refrain creating a function inside another function.
Refer to the coding guidelines below:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR0AgTGRhVpMnMrBg9rIM05xqWwT9L0CCuzzuA_Yri3u3BW5iuTvvRNhkN0_aem_AeZMm9DFw7FXEQYpH4WcqolKKh_OZYLYbQrP18QItx6hu_p7pGos7XXWnUIFlXcqBpk_A6mKXBQ_SAx2CynH7k15#slide=id.g2651733a4ec_0_9